### PR TITLE
Rename whohacks to whois.

### PR DESCRIPTION
**Pull request by EduKav1813**
The project is named "whohacks" while the actual
folder is named "whois", which causes `poetry
install` to fail.

### CODE REVIEW

**Review by not7cd — CHANGES_REQUESTED** [review]
I think there should be a different setting for changing the module name that is being found by poetry.

**Comment by EduKav1813** [comment]
Ok, I will have a look

**Comment by EduKav1813** [comment]
I managed to fix this problem by using the "packages" section and modifying the Dockerfile. I also moved from using the deprecated `poetry install --no-dev` to newer `poetry install --only main.`

File: Dockerfile [hunk 1]
```diff
@@ -7,9 +7,10 @@ RUN poetry config virtualenvs.create false
 
 COPY pyproject.toml /app/
 COPY poetry.lock /app/
+COPY whois /app/whois/
```
Comment (not7cd) [inline Dockerfile:10]: You shouldn't be required to do this. We are not copying project files before requirement install, so every code change doesn't force requirement installation.

**Comment by EduKav1813** [comment]
Closed in favor of https://github.com/hspsh/whohacks/pull/81

**Pull request closed without merging**